.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,11 +16,12 @@ For example:
 -->
 
 ## Change have been demonstrated
-
-Changes to the user interface or content should be demonstrated to Content Design and Interaction Design before being merged. This is to ensure they can make any necessary changes to Figma.
+Changes to the user interface or content should be demonstrated to Content Design and Interaction Design before being merged. This is to ensure they are aware of changes and behaviours (for example, how error screens appear and whether invalid entries can be amended), and can make any necessary changes to Figma.
 
 - [ ] Changes to the user interface have been demonstrated
 
+<!-- Include screenshots where possible, including those representing error states. -->
```

Comment:
Wondering if here we want to suggest ways of including screenshots. For example, I find either [tables](https://github.com/govuk-one-login/authentication-frontend/pull/1187) or the dropdown thing that you do more useful than a wall of screenshots

Reply:
Thanks @BeckaL - that's a great suggestion 👍🏽 . I've just pushed an update to reference these same two example PRs.